### PR TITLE
update contributor quickstart in readme and pull request template. re…

### LINKED ISSUE #252: [Enhancement] Update ReadMe and Contributing files to make onboarding smoother
### Description
While previous improvements made it easier for new team members to get up and running, the current ReadMe and Contributing docs are still a little long-winded and not super easy to navigate. The Contributing.md file is outdated at this point and may over-document the process to the point of being overwhelming/confusing. 

### Recommended updates
- [ ] Consider adding the basic steps for installing Docker Compose and/or link to https://docs.docker.com/compose/install/
- [ ] Make "Browse the client at lvh.me, ping the api at api.lvh.me." more explicit. I.e. "View the front-end in your browser at lvh.me" etc.
- [ ] Explain "Services" section more explicitly in plain language
- [ ] Add a reference to the API's swagger docs (?)
  - `api.lvh.me/swagger` - TY 
- [ ] Condense the workflow description and include it in the ReadMe, eliminating the need for a separate doc.
- [ ] Correct header hierarchy for easier scanning/visual simplicity
- [ ] Update technologies list - TY
  - Zurb -> Semantic UI
  - TSLint -> ESLint (Pending PR #251 / Issue #250 )
  - Black -> Doesn't seem to actually exist in the code? Also, I prefer autopep8. Black can sometimes lead to obnoxious line formatting.

### Good examples worth borrowing from
- https://github.com/openoakland/openbudgetoakland

PR #253 -TY

### PR DESCRIPTION
**Pull request by TangoYankee**
…move contributing doc

---
name: Pull Request
about: Request merging changes into the code base
title: ''
labels: ''
reviewers: ''

---

## Checklist
- [ ] Add description
- [ ] Reference open issue pull request addresses
- [ ] Pass functional tests
  - spin up docker container service `docker up -d --build`
  - enter api container `docker-compose exec api /bin/bash`
  - run api tests `make test`
  - exit container `ctrl/command+D` or `exit`
  - enter web container `docker-compose exec web /bin/sh`
  - run front-end tests `npm test` or `jest`
  - lint `npm run lint` or `tslint -p .` with optional `--fix` 
  - exit container `ctrl/command+D` or `exit`
- [ ] Request code review
  - Please allow **36 hours** from opening a pull request before merging a pull request- even if it has already recieved an approving review.
- [ ] Address comments on code and resolve requested changes
- [ ] Merge own code

## Description
Issue: #252 

*Brief description of solution*
@Jess I made the majority of changes I wanted to see for the developer portion of onboarding. If you want, you can checkout this branch and modify it as you see fit. I do plan on opening an Issue for seeding the pollutant, as you'll see referenced in the README.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,26 +1,17 @@
...
-reviewers: ''
-
----
-
 ## Checklist
 - [ ] Add description
-- [ ] Reference open issue pull request addresses
-- [ ] Pass functional tests
-  - spin up docker container service `docker up -d --build`
+- [ ] Reference the open issue that pull the request addresses
```
Comment (theecrit) [inline .github/pull_request_template.md]: Typo: _that the pull request_

File: README.md [hunk 2]
```diff
@@ -21,27 +21,32 @@ To participate in this project, we ask you to abide by the OpenOakland [Code of
...
+- woaq-proxy: [nginx-proxy](https://github.com/jwilder/nginx-proxy), [ssl-autogen](https://github.com/JrCs/docker-letsencrypt-nginx-proxy-companion)   
 - woaq-web:
-[typescript](https://www.typescriptlang.org/), [react](https://reactjs.org/), [razzle](https://github.com/jaredpalmer/razzle), [zurb](https://react.foundation/)
-
+[typescript](https://www.typescriptlang.org/), [react](https://reactjs.org/), [razzle](https://github.com/jaredpalmer/razzle), [semantic UI](https://react.semantic-ui.com/)  
 - woaq-api: [python](https://www.python.org/), [django-rest-framework](https://www.django-rest-framework.org/)
-
 - woaq-db: [postgis](https://postgis.net/)
 
+## Domains
```
Comment (theecrit) [inline README.md]: Can we add some plain language intro, like `You can now access the services at the following domains in your browser:`.

Just trying to make this a little friendlier for folks still on a learning curve. :)

File: README.md [hunk 3]
```diff
@@ -21,27 +21,32 @@ To participate in this project, we ask you to abide by the OpenOakland [Code of
...
 - woaq-web:
-[typescript](https://www.typescriptlang.org/), [react](https://reactjs.org/), [razzle](https://github.com/jaredpalmer/razzle), [zurb](https://react.foundation/)
-
+[typescript](https://www.typescriptlang.org/), [react](https://reactjs.org/), [razzle](https://github.com/jaredpalmer/razzle), [semantic UI](https://react.semantic-ui.com/)  
 - woaq-api: [python](https://www.python.org/), [django-rest-framework](https://www.django-rest-framework.org/)
-
 - woaq-db: [postgis](https://postgis.net/)
 
+## Domains 
+- Webpage: [lvh.me](http://lvh.me)
```
Comment (theecrit) [inline README.md]: How about `Homepage` instead of Webpage (more specific)? Yes, I'm being nitpicky.

File: README.md [hunk 4]
```diff
@@ -21,27 +21,32 @@ To participate in this project, we ask you to abide by the OpenOakland [Code of
...
+- API: [api.lvh.me](http://api.lvh.me)
+- API Documentation: [api.lvh.me/swagger](http://api.lvh.me/swagger)
+
+## Code Quality Checks
+
+Follow testing and linting instructions in the [Pull Request Template](/.github/pull_request_template.md)
+
 ## Contributing workflow and guidelines
 
 See the [Contributing](https://github.com/openoakland/woeip/blob/master/.github/contributing.md) section.
```
Comment (theecrit) [inline README.md]: Am I misreading? It looks like the contributing.md was deleted?

**Review by theecrit — CHANGES_REQUESTED** [review]
Just made some minor notes re: clarity.

File: README.md (hunk 4 above)
Comment (TangoYankee) [reply README.md]: It was, as I understood the request to [condense the developer setup.](https://github.com/openoakland/woeip/issues/252#issuecomment-679157864) No te gusta? 

...Which line 52 should be deleted, if you do want the contributing doc gone.

File: .github/pull_request_template.md (hunk 1 above)
Comment (TangoYankee) [reply .github/pull_request_template.md]: D'oh

File: README.md (hunk 3 above)
Comment (TangoYankee) [reply README.md]: I see what you mean. And it is language I've been struggling to pin down. Really, I'm trying to highlight that it's frontend v. backend v. api documentation.

File: README.md (hunk 2 above)
Comment (TangoYankee) [reply README.md]: I don't see why not

File: README.md (hunk 3 above)
Comment (theecrit) [reply README.md]: In that case, it's okay to over-clarify, I think. For example:

> _Front-end:_
> - Homepage (will eventually be woaq.org): lvh.me
> 
> _Back-end:_
> - Datavase: <link>
> - API: <link>

Comment (theecrit) [reply README.md]: No right answer, here!

File: README.md (hunk 4 above)
Comment (theecrit) [reply README.md]: Yes, that's what I was getting at: line 52 references a deleted file so should come out.

**Pull request merged**